_includes/releases/v22.1/v22.1.19.md

```diff
@@ -0,0 +1,62 @@
+## v22.1.19
+
+Release Date: April 24, 2023
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-19-general-changes">General changes</h3>
+
+- Queries with invalid syntax are now logged at the `INFO` level on the [`SQL_EXEC` logging channel](../v22.1/sql-audit-logging.html). Previously, they were logged at the `ERROR` level. [#101090][#101090]
+
+<h3 id="v22-1-19-sql-language-changes">SQL language changes</h3>
+
+- Added the `prepared_statements_cache_size` [session setting](../v22.1/set-vars.html) which helps to prevent [prepared statement](../v22.1/savepoint.html#savepoints-and-prepared-statements) leaks by automatically deallocating the least-recently-used prepared statements when the cache reaches a given size. [#99264][#99264]
+
+<h3 id="v22-1-19-db-console-changes">DB Console changes</h3>
+
+- New data is now auto-fetched every 5 mins on the [**Statement and Transaction Fingerprints**](../v22.1/ui-statements-page.html#statement-fingerprint-page) pages. [#100702][#100702]
+
+<h3 id="v22-1-19-bug-fixes">Bug fixes</h3>
+
+- Previously, [`ADD COLUMN ... DEFAULT cluster_logical_timestamp()`](../v22.1/alter-table.html) would crash the node and leave the table in a corrupt state. The root cause is a `nil` pointer dereference. The bug is now fixed by returning an unimplemented error and hence disallowing using the [builtin function](../v22.1/functions-and-operators.html#array-functions) as the default value when backfilling. [#99682][#99682]
+- Fixed a bug where the stats columns on the [**Transaction Fingerprint Overview**](../v22.1/ui-transactions-page.html#transaction-details-page) page was continuously incrementing. [#99405][#99405]
+- Fixed a bug that prevented the garbage-collection job for the [`TRUNCATE TABLE`](../v22.1/truncate.html) command from successfully finishing if the table descriptor had already been garbage-collected. The garbage-collection job now succeeds in this situation by handling the missing descriptor edge case. [#100146][#100146]
```

Comment:
Nit: "garbage-collection job" doesn't need to be hyphenated here as "garbage collection" is a well-known term on its own. (And remove the hyphen from all other instances in this bullet point.)

We can also link the Garbage Collection section here, on the first instance /architecture/storage-layer.html#garbage-collection